custom_components/tuya_local/devices/solareast_heatpump_adlarcastra.yaml

```diff
@@ -0,0 +1,401 @@
+name: Heat pump Adlår Castra / SolarEast
+products:
+  - id: g9m7honkxjweukvt
+    manufacturer: SolarEast
+    model: Aurora II
+    name: Adlår Castra Aurora II 6kWh
+entities:
+  - entity: switch
+    translation_key: switch_x
+    translation_placeholders:
+      x: ""
+    class: switch
+    category: config
+    dps:
+      - id: 1
+        name: switch
+        type: boolean
+  - entity: select
+    name: mode
+    category: config
+    dps:
+      - id: 2
+        name: option
+        type: string
+        mapping:
+          - dps_val: "cold"
+            value: "cold"
+          - dps_val: "heating"
+            value: "heating"
+          - dps_val: "floor_heating"
+            value: "floor_heating"
+          - dps_val: "hot_water"
+            value: "hot_water"
+          - dps_val: "cold_and_hotwater"
+            value: "cold_and_hotwater"
+          - dps_val: "heating_and_hot_water"
+            value: "heating_and_hot_water"
+          - dps_val: "floor_heating_and_hot_water"
+            value: "floor_heating_and_hot_water"
+  - entity: number
+    name: temp_set
+    category: config
+    dps:
+      - id: 4
+        name: value
+        type: integer
+        unit: "C"
+        range:
+          min: 5
+          max: 75
+        step: 1
```

Comment:
`step` belongs inside a `mapping`, but in the case of `step: 1` it is not required and can just be deleted.